// @uiowa/spinner/esm2020/lib/spinner.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { ActionSpinnerComponent } from './action-spinner/action-spinner.component';
import { LoadingBarComponent } from './loading-bar/loading-bar.component';
import { UiowaRingComponent } from './uiowa-ring/uiowa-ring.component';
import { LoadingPlaceholderComponent } from './loading-placeholder/loading-placeholder.component';
import * as i0 from "@angular/core";
export class SpinnerModule {
}
SpinnerModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.11", ngImport: i0, type: SpinnerModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
SpinnerModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.11", ngImport: i0, type: SpinnerModule, declarations: [ActionSpinnerComponent,
        LoadingBarComponent,
        UiowaRingComponent,
        LoadingPlaceholderComponent], imports: [CommonModule], exports: [ActionSpinnerComponent,
        LoadingBarComponent,
        UiowaRingComponent,
        LoadingPlaceholderComponent] });
SpinnerModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.11", ngImport: i0, type: SpinnerModule, imports: [[CommonModule]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.11", ngImport: i0, type: SpinnerModule, decorators: [{
            type: NgModule,
            args: [{
                    imports: [CommonModule],
                    declarations: [
                        ActionSpinnerComponent,
                        LoadingBarComponent,
                        UiowaRingComponent,
                        LoadingPlaceholderComponent,
                    ],
                    exports: [
                        ActionSpinnerComponent,
                        LoadingBarComponent,
                        UiowaRingComponent,
                        LoadingPlaceholderComponent,
                    ],
                }]
        }] });
